Test PgBouncer with pytest failed

Hello Team, I had a problem when running `pytest`.

```
[root@08eea0aa1272 pgbouncer]# pytest
ImportError while loading conftest '/tmp/pgbouncer/test/conftest.py'.
test/conftest.py:7: in <module>
    from .utils import (
test/utils.py:9: in <module>
    from contextlib2 import asynccontextmanager
E   ModuleNotFoundError: No module named 'contextlib2'
```

I follow the steps in https://github.com/pgbouncer/pgbouncer/blob/master/test/README.md.

```
[root@08eea0aa1272 pgbouncer]# python3 --version
Python 3.6.8
```

```
[root@08eea0aa1272 pgbouncer]# pip3 install -r requirements.txt
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Requirement already satisfied: pytest in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 1))
Requirement already satisfied: pytest-asyncio in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 2))
Requirement already satisfied: pytest-timeout in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 3))
Requirement already satisfied: pytest-xdist in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 4))
Requirement already satisfied: psycopg in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 5))
Requirement already satisfied: filelock in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 6))
Requirement already satisfied: tomli>=1.0.0 in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: attrs>=19.2.0 in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: importlib-metadata>=0.12; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: py>=1.8.2 in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: pluggy<2.0,>=0.12 in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: iniconfig in /usr/local/lib/python3.6/site-packages (from pytest->-r requirements.txt (line 1))
Requirement already satisfied: execnet>=1.1 in /usr/local/lib/python3.6/site-packages (from pytest-xdist->-r requirements.txt (line 4))
Requirement already satisfied: backports.zoneinfo>=0.2.0; python_version < "3.9" in /usr/local/lib64/python3.6/site-packages (from psycopg->-r requirements.txt (line 5))
Requirement already satisfied: typing-extensions>=3.10; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from psycopg->-r requirements.txt (line 5))
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging->pytest->-r requirements.txt (line 1))
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->pytest->-r requirements.txt (line 1))
Requirement already satisfied: importlib_resources in /usr/local/lib/python3.6/site-packages (from backports.zoneinfo>=0.2.0; python_version < "3.9"->psycopg->-r requirements.txt (line 5))
```

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -73,7 +73,6 @@ task:
     - apt-get -y install $pkgs
     - python3 -m venv /venv
     - /venv/bin/pip install -r requirements.txt
-    - if [ "$(lsb_release -cs)" = "bionic" ]; then /venv/bin/pip install contextlib2; fi
     - useradd user
     - chown -R user .
     - echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
@@ -114,7 +113,6 @@ task:
     - tar xvzf /tmp/pandoc.tar.gz --strip-components 1 -C /usr/local/
 # XXX: python too old
     - if cat /etc/centos-release | grep -q ' 7'; then true; else pip3 install -r requirements.txt; fi
-    - if cat /etc/centos-release | grep -q ' 7'; then true; else pip3 install contextlib2; fi
     - useradd user
     - chown -R user .
     - echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,3 +4,4 @@ pytest-timeout
 pytest-xdist
 psycopg
 filelock
+contextlib2; python_version < "3.7"

